src/chain/chain.rs

```diff
@@ -369,11 +369,15 @@ impl Chain {
     // Increment the age counters of the members.
     pub fn increment_age_counters(&mut self, trigger_node: &PublicId) {
         let our_section_size = self.state.our_joined_members().count();
-        let safe_section_size = self.safe_section_size();
+        let our_prefix = *self.state.our_prefix();
+
+        // Is network startup in progress?
+        let startup =
+            our_prefix == Prefix::default() && our_section_size < self.safe_section_size();
 
         // As a measure against sybil attacks, we don't increment the age counters on infant churn
-        // once we reach safe_section_size.
-        if our_section_size >= safe_section_size
+        // once we are past the startup phase.
```

Comment:
Once we completed the startup phase, and ... ?

Reply:
Sorry, not sure what are you trying to say. The complete sentence is 

> As a measure against sybil attacks, we don't increment the age counters on infant churn once we are past the startup phase.

is there something wrong with it?

Reply:
Ah, missed the above sentence.

However, still think:
once we are past the startup phase => once we completed the startup phase
will be better?

Reply:
Ah, missed the above sentence.

However, still think:
once we are past the startup phase => once we completed the startup phase
will be better?